octoprint_discordremote/__init__.py

```diff
@@ -36,6 +36,17 @@ class DiscordRemotePlugin(octoprint.plugin.EventHandlerPlugin,
                           octoprint.plugin.TemplatePlugin,
                           octoprint.plugin.ProgressPlugin,
                           octoprint.plugin.SimpleApiPlugin):
+  
+    @property
```

Comment:
What is the purpose of these changes?

Reply:
That's a leftover from when I stored the file type list as a variable instead of a function and needed a getter, I'll remove it.

Reply:
Ha, it just occurred to me that you might be talking about the entire bunch of lines changed, not just the @property decorator.
I had to add all of the zip filestpyes to Octoprint's list of allowed files or else Octoprint would instantly delete all of them upon receiving them:

allowed(self) is a function that provides a string containing a list of all of the zip filestpyes (.zip, .zip.001, .zip.002 etc)
get_extension_tree(self, *args, **kwargs) then adds that list to Octoprint's file types list.

I guess this could be written in a more elegant way, either by just building the list inside get_extension_tree (if the function is called only once) or building the list inside __init__ and storing it as a variable (if get_extension_tree is called multiple times).

Reply:
Cool, that makes a lot of sense, mayube chuck a comment in and its all good

Reply:
With Commit 3d1e4b9 I have added a comment and simplified the function a bit - it now doesn't build a string and turns it into a list, it builds the light right away.